mdx_oembed/endpoints.py

```diff
@@ -6,6 +6,14 @@
     'https?://youtu.be/*',
 ])
 
+SLIDESHARE = oembed.OEmbedEndpoint('http://www.slideshare.net/api/oembed/2', [
+    'http://www.slideshare.net/*/*',
```

Comment:
I agree putting a regep looks simpler. however these are the fixed subdomain list which is being used. I wanted to keep it tight to exactly the specifics (based on http://oembed.com/#section5 details of oembed for slideshare). If you feel we can put something like ```http://*.slideshare.net/*/*``` 

Also I realized ```http://oembed.com/providers.json``` gives a nice feed of various providers why don't we source from there.